notes/force-move-implementation.md

```diff
@@ -121,11 +130,11 @@ From this we can say the following about the way we chose to hash states:
 
 ### Proposal for state hash
 
-The hash of the state that the participants sign should be the hash of the following:
+The data that the participants sign should be the hash of the following:
 
 - TurnNum
 - isFinal
-- ChannelId
+- ChannelId (= fixedPartHash)
```

Comment:
I think there's a slight distinction here. `AppDefinition` should be part of the "fixed part" (in terms of the data passed in) but not in terms of the hash. I guess it might be worth avoiding the terms "fixed" and "variable" in the hash to avoid confusion.

Reply:
OK I have moved it in 4de94bf